app/views/users/login_form.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>User Log In</h1>
+
+<% form_with model: @user, url: login_path do |f| %>
+  <div class="form-group">
+    <%= f.label :name %>
+    <%= f.text_field :name, placeholder: "Username", class:"form-control" %>
```

Comment:
Line 3 should be a value tag (`<%=`), otherwise the form won't appear on the page.